### ISSUE TITLE
intellisense for Razor View has lost after upgrade to csproj

### ISSUE THREAD

**Issue by thiennn** [simplcommerce/SimplCommerce#136]
In csproj, it haven't been supported yet. 
https://github.com/aspnet/Mvc/issues/5975

**Comment by MaherJendoubi** [comment]
@thiennn You have to install Razor Language Services extension https://marketplace.visualstudio.com/items?itemName=ms-madsk.RazorLanguageServices

**Comment by MaherJendoubi** [comment]
When you’re building ASP.NET Core applications using the MVC pattern, it would be nice to have some assistance when writing your views in razor templates.  The Razor Language Services extension gives you IntelliSense for .NET expressions, hover tooltips for elements, and syntax highlighting for tag helpers.

**Comment by MaherJendoubi** [comment]
![image](https://cloud.githubusercontent.com/assets/1798510/24194121/1990ac26-0ef5-11e7-8b9f-e53813be4d60.png)

**Comment by MaherJendoubi** [comment]
@thiennn be careful when I installed this extension my VS2017 becomes instable :( and it crashes when opening a razor view file!

**Comment by MaherJendoubi** [comment]
@thiennn https://github.com/aspnet/Tooling/issues/1022#issuecomment-288473579

**Comment by thiennn** [comment]
waiting on this https://github.com/aspnet/Tooling/issues/1013
